### ISSUE TITLE
Forecast attribute of WeatherEntity is deprecated - Needs fixing

### ISSUE THREAD

**Issue by tommyjlong** [tommyjlong/wral_weather#4]
As of 2023.9, Home Assistant is deprecating "forecast" as an attribute.
A Log warns of this: 
```
custom_components.wral_weather.weather::None
 is using a forecast attribute on an instance of WeatherEntity, this is deprecated and will be unsupported from Home Assista
nt 2024.3. Please report it to the custom integration author.
```

**Comment by tommyjlong** [comment]
This was fixed in version 2.0.3

**Comment by tommyjlong** [comment]
closing
